packages/release-brancher/src/release-brancher.ts

```diff
@@ -328,7 +335,10 @@ export class Runner {
         }
       }
     }
-    const message = 'feat: configure initial sp version';
+    const message =
+      this.releaseType === 'java-lts'
+        ? 'feat: configure initial sp version'
+        : 'feat: configure the protected release branch';
```

Comment:
For workflow branch pull request, the message does not carry "sp" when specified with "java-lts-no-sp" releaes type.